### PR TITLE
Add project description

### PR DESCRIPTION
**Pull request by ep-cc**
Add project description to Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# git-refresh-240725
\ No newline at end of file
+# Number guessing game
+
+## Tech stack
+- HTML
```
Comment (tloszkov) [inline README.md:4]: Html???

Comment (ep-cc) [reply README.md:4]: Hyper Text Markup Language

**Review by gakalmar — CHANGES_REQUESTED** [review]
Please use another language

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+- CSS
```
Comment (tloszkov) [inline README.md:5]: Change to Tailwind!

File: README.md [hunk 3]
```diff
@@ -1 +1,7 @@
...
+- Javascript
```
[the remark this answers is not in the document]
Comment (ep-cc) [reply README.md:6]: What do you think Typescript?

[the remark this answers is not in the document]
Comment (ep-cc) [reply README.md]: Done

**Pull request merged**